pkg/controllers/work/apply_controller.go

```diff
@@ -50,6 +52,8 @@ import (
 
 const (
 	workFieldManagerName = "work-api-agent"
+	fleetSystemNamespace = "fleet-system"
```

Comment:
I think it's time for us to create a constants package. We avoided it during the reserved namespace work as we used a prefix. But my current PR is using a string literal for an ObjectKey. 

https://github.com/Azure/fleet/pull/346/files#diff-df64e37f2222b761c62bd8bf32c0347aae0fe1f46ff6acf0e66c2b61bd30d761R398